tests/test_e2e.py

```diff
@@ -336,67 +342,31 @@ def get_lab_user_secret(cls, lab_name="msidlab4"):
     def get_lab_user(cls, **query):  # https://docs.msidlab.com/labapi/userapi.html
         resp = cls.session.get("https://msidlab.com/api/user", params=query)
         result = resp.json()[0]
+        authority_base = "https://login.microsoftonline.com/"
+        graph_endpoint = "https://graph.microsoft.com/.default"
+        if "azureenvironment" in query and query["azureenvironment"] == "azureusgovernment":
+            authority_base = "https://login.microsoftonline.us/"
+            graph_endpoint = "https://graph.microsoft.us/.default"
```

Comment:
Ideally we should avoid "assign a default value; and then change it immediately" pattern. The following approach is more bullet-proof, and future-proof.

```suggestion
        _env = query.get("azureenvironment", "").lower()
        base = {
            "azureusgovernment": "https://login.microsoftonline.us",
            }.get(_env, "https://login.microsoftonline.com")
        scopes = {
            "azureusgovernment": ["https://graph.microsoft.us/.default"],
            }.get(_env, ["https://graph.microsoft.com/.default"])
```

You would also need to adjust the rest of this function accordingly.